code/src/App.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react'
 
+import { Form } from './components/Form'
+import { HappyThoughts } from './components/HappyThoughts'
+
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    
+    <section className='app-content'>
+      <Form  />
+      <HappyThoughts />
+    </section>
+  );
```

Comment:
Looks like you've chosen not to use semicolons in your JavaScript, be consistent and remove the ones on lines 13 and 14. It may also help remove the extra one showing at the bottom of the page. Not sure where that one is from exactly. 
<img width="381" alt="Screenshot 2020-11-04 at 3 18 53 PM" src="https://user-images.githubusercontent.com/64912576/98123047-a9b22080-1eb1-11eb-9cc5-635b82e9eace.png">